pep-0567.rst

```diff
@@ -551,6 +604,53 @@ code unmodified, but will automatically enable support for
 asynchronous code.
 
 
+Examples
+========
+
+Converting code that uses threading.local()
+-------------------------------------------
+
+A typical code that uses ``threading.local()`` usually looks like
+the following snippet::
+
+    class mylocal(threading.local):
+        # Subclass threading.local to specify a default value.
+        value = 'spam'
+
+    mylocal = mylocal()
+
+    # To set a new value:
+    mylocal.value = 'new value'
+
+    # To read the current value:
+    mylocal.value
```

Comment:
Assign it to a variable or use it in a larger expression.

Reply:
fixed